doc/BUILD-Debian-LABO-v2.md

```diff
@@ -1,117 +0,0 @@
-TODO
-
-1. install boost (apt-get)
-2. install boost numeric bindings
```

Comment:
not anymore:
- boost numeric bindings
-  gfortran
- glog
- SuiteSparse
- optpp
- Ceres Solver